### ISSUE TITLE
pachd pods logging file contents when pipelines triggered 

### ISSUE THREAD

**Issue by josibake** [pachyderm/pachyderm#1309]
as pipeline processes data, it appears to also log contents to pachd pods, causing pods to run out of memory. During pipeline, running `kubectl logs po/pachd-<id>` gives:

```2017-02-07T05:08:26Z INFO  protorpclog.Call {"service":"pps.API","method":"CreatePipeline","request":"pipeline:\u003cname:\"get_urls\" \u003e transform:\u003cimage:\"josibake/get-urls\" cmd:\"/pfs/events_raw/events-2017-02-01.log\" cmd:\"/pfs/out\" \u003e parallelism_spec:\u003cconstant:1 \u003e inputs:\u003crepo:\u003cname:\"events_raw\" \u003e method:\u003cincremental:DIFF \u003e \u003e ","duration":"0.062141109s"}
2017-02-07T05:08:33Z INFO  protorpclog.Call {"service":"CacheServer","method":"Get","request":"group:\"block\" key:\"l4k1SQIYcqjkcDUWnW3bdgDYv5-myciguOeb-0f2to9W3eoOgvtLaRRpX_1Xtzu5ZW1GR6_nRxWF9v8X_R61Mg==\" ","duration":"0.000s"}
2017-02-07T05:08:33Z INFO  protorpclog.Call {"service":"CacheServer","method":"Get","request":"group:\"block\" key:\"l4k1SQIYcqjkcDUWnW3bdgDYv5-myciguOeb-0f2to9W3eoOgvtLaRRpX_1Xtzu5ZW1GR6_nRxWF9v8X_R61Mg==\" ","duration":"0.246358315s"}
2017-02-07T05:08:33Z INFO  protorpclog.Call {"service":"groupcachepb.GroupCache","method":"Get","request":"group:\"block\" key:\"l4k1SQIYcqjkcDUWnW3bdgDYv5-myciguOeb-0f2to9W3eoOgvtLaRRpX_1Xtzu5ZW1GR6_nRxWF9v8X_R61Mg==\" ","response":"value:\"{\\\"@version\\\":\\\"1\\\",\\\"@timestamp....```

**Comment by derekchiang** [comment]
@jdoliner any idea what's logging the third line here?  I seem to remember that you mentioned that there's some protobuf magic that automatically logs requests/responses which is probably what's happening here.

**Comment by jdoliner** [comment]
@derekchiang third line is being logged by the caching server, there is some protobuf magic that automatically logs things (technically grpc magic). However that magic shouldn't be present anymore in master since we have a more vanilla usage of protobuf now. I'll test this in master but I'm pretty sure it shouldn't be an issue anymore.

@josibake which version of Pachyderm did you encounter this on?

**Comment by josibake** [comment]
@jdoliner currently running v1.3.4

**Comment by jdoliner** [comment]
This is fixed in master and v1.3.5 by the grpc upgrade. It was caused by one of the wrapper libraries we were using.
